Python/PyPlasma.h

```diff
@@ -26,7 +26,8 @@
 PyObject* PyString_FromSTString(const ST::string& str);
 PyObject* PyUnicode_FromSTString(const ST::string& str);
 ST::string PyAnyString_AsSTString(PyObject* str);
-#define PyAnyString_Check(ob) (PyUnicode_Check(value) || PyBytes_Check(value))
+int PyAnyString_PathDecoder(PyObject* obj, ST::string* str);
+#define PyAnyString_Check(ob) (PyUnicode_Check(ob) || PyBytes_Check(ob))
```

Comment:
Heh, good catch

Reply:
This was one of those, "wait, what's going on here..." moments 😄